collector/fixtures/e2e-output.txt

```diff
@@ -785,43 +788,82 @@ node_md_blocks_synced{device="md10"} 3.14159265e+08
 node_md_blocks_synced{device="md11"} 4.190208e+06
 node_md_blocks_synced{device="md12"} 3.886394368e+09
 node_md_blocks_synced{device="md127"} 3.12319552e+08
+node_md_blocks_synced{device="md13"} 4.88383936e+08
+node_md_blocks_synced{device="md20"} 9.76751616e+08
 node_md_blocks_synced{device="md219"} 7932
 node_md_blocks_synced{device="md3"} 5.853468288e+09
 node_md_blocks_synced{device="md4"} 4.883648e+06
+node_md_blocks_synced{device="md50"} 8.790400512e+09
 node_md_blocks_synced{device="md6"} 1.6775552e+07
 node_md_blocks_synced{device="md7"} 7.813735424e+09
 node_md_blocks_synced{device="md8"} 1.6775552e+07
 node_md_blocks_synced{device="md9"} 523968
-# HELP node_md_disks Total number of disks of device.
+# HELP node_md_disks Number of disks of device. (state="any" equals to the number of disks in the array)
```

Comment:
In that case I'm not sure it makes sense to use labels here as @SuperQ proposed. Including a total/any state as a label is something we have to avoid, a `sum()` across all labels must be a meaningful total. If we can't do that, we'll have to resort to individual metrics.